backend/src/main/java/net/es/oscars/sb/nso/NsoLspStateSyncer.java

```diff
@@ -242,6 +257,11 @@ public boolean sync(String path, boolean dryRun) throws NsoStateSyncerException
                     NsoServicesWrapper.NsoServicesWrapperBuilder addBuilder = NsoServicesWrapper.builder();
                     List<NsoLSP> addList = new ArrayList<>();
                     NsoLSP lsp = wrapper.getInstance();
+                    if (!this.isOscarsManaged(lsp)) {
+                        log.error("attempted to add an unmanaged LSP " + lsp.getName() + "/" + lsp.getDevice());
```

Comment:
same here... can we make this a proper error that must be handled, or switch to a warning?